output/index.go

```diff
@@ -3,12 +3,14 @@ package output
 
 import (
 	"github.com/relex/slog-agent/base/bconfig"
+	"github.com/relex/slog-agent/output/datadog"
 	"github.com/relex/slog-agent/output/fluentdforward"
 )
 
 func init() {
 	bconfig.RegisterConfigConstructors(bconfig.LogOutputConfigCreatorTable{
 		"fluentdForward": func() bconfig.LogOutputConfig { return &fluentdforward.Config{} },
+		"datadog":        func() bconfig.LogOutputConfig { return &datadog.Config{} },
```

Comment:
can be moved up one line to keep the list in alphabet order, if no other rule applies.